docs/conf.py

```diff
@@ -169,8 +169,11 @@
 
 # -- Turn on nitpicky mode for sphinx (to warn about references not found) ----
 #
-# nitpicky = True
-# nitpick_ignore = []
+nitpicky = True
+nitpick_ignore = [
+    ("py:class", "specutils.spectra.spectrum1d.Spectrum1D"),
+    ("py:obj", "specutils.Spectrum1D")
```

Comment:
Why are these ignored?

Reply:
because of what i mentioned in my comments. for some reason, sphinx isn't resolving it. this is the solution i found via posts by you and erik bray...